samples/sqlservices/src/controllers/mainController.ts

```diff
@@ -701,9 +708,150 @@ export default class MainController implements vscode.Disposable {
 		await view.initializeModel(formWrapper);
 	}
 
-	//#endregion
+	private async getGraphTabContent(view: azdata.ModelView): Promise<void> {
+		const barChart = view.modelBuilder.chart<'bar', azdata.BarChartData, azdata.BarChartOptions>()
+			.withProps({
+				chartId: 'barChart1',
+				chartType: 'bar',
+				data: chartExamples.barData,
+				options: chartExamples.barOptions,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const horizontalBarChart = view.modelBuilder.chart<'horizontalBar', azdata.HorizontalBarChartData, azdata.HorizontalBarChartOptions>()
+			.withProps({
+				chartId: 'horizBarChart1',
+				chartType: 'horizontalBar',
+				data: chartExamples.horizontalBarData,
+				options: chartExamples.horizontalBarOptions,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const lineChart = view.modelBuilder.chart<'line', azdata.LineChartData, azdata.LineChartOptions>()
+			.withProps({
+				chartId: 'lineChart1',
+				chartType: 'line',
+				data: chartExamples.lineData,
+				options: chartExamples.lineOptions,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const pieChart = view.modelBuilder.chart<'pie', azdata.PieChartData, azdata.PieChartOptions>()
+			.withProps({
+				chartId: 'pieChart1',
+				chartType: 'pie',
+				data: chartExamples.pieData,
+				options: chartExamples.pieOptions,
+				width: '300px',
+				height: '300px'
+			}).component();
+
+		const doughnutChart = view.modelBuilder.chart<'doughnut', azdata.DoughnutChartData, azdata.DoughnutChartOptions>()
+			.withProps({
+				chartId: 'doughnutChart1',
+				chartType: 'doughnut',
+				data: chartExamples.doughnutData,
+				options: chartExamples.doughnutOptions,
+				width: '400px',
+				height: '400px'
+			}).component();
+
+		const scatterplot = view.modelBuilder.chart<'scatter', azdata.ScatterplotData, azdata.ScatterplotOptions>()
+			.withProps({
+				chartId: 'scatterplot1',
+				chartType: 'scatter',
+				data: chartExamples.scatterData,
+				options: chartExamples.scatterOptions,
+				width: '400px',
+				height: '400px'
+			}).component();
+
+		const bubbleChart = view.modelBuilder.chart<'bubble', azdata.BubbleChartData, azdata.BubbleChartOptions>()
+			.withProps({
+				chartId: 'bubbleChart1',
+				chartType: 'bubble',
+				data: chartExamples.bubbleData,
+				options: chartExamples.bubbleOptions,
+				width: '500px',
+				height: '500px'
+			}).component();
 
-	//#region Registrations
+		const polarChart = view.modelBuilder.chart<'polarArea', azdata.PolarAreaChartData, azdata.PolarAreaChartOptions>()
+			.withProps({
+				chartId: 'polarChart1',
+				chartType: 'polarArea',
+				data: chartExamples.polarData,
+				options: chartExamples.polarOptions,
+				width: '500px',
+				height: '500px'
+			}).component();
+
+		const radarChart = view.modelBuilder.chart<'radar', azdata.RadarChartData, azdata.RadarChartOptions>()
+			.withProps({
+				chartId: 'radarChart1',
+				chartType: 'radar',
```

Comment:
I agree.  I've spent lots of time fighting with Typescript's ability to give compile-time guarantees, which has resulted in some redundancy, but I'll see if I trim it down further.

Reply:
Definitely don't spend too much time on it, we can always come back later and fix it up if it ends up being too clunky.

Reply:
I can restrict this property to only the correct value, but I don't think I can eliminate it entirely since the `chart` object type parameters are lost at compilation time, and I need this value at runtime.